below/render/src/default_configs.rs

```diff
@@ -276,6 +282,17 @@ impl HasRenderConfig for model::CgroupCpuModel {
     }
 }
 
+impl HasRenderConfig for model::CgroupPidsModel {
+    fn get_render_config_builder(field_id: &Self::FieldId) -> RenderConfigBuilder {
+        use model::CgroupPidsModelFieldId::*;
+        let rc = RenderConfigBuilder::new();
+        match field_id {
+            PidsCurrent => rc.title("Pids Current").format(Precision(1)),
+            PidsMax => rc.title("Pids Max").format(Precision(1)),
```

Comment:
This doesn't handle the -1 case that represents `max`. See `MaxOrReadableSize` for example. Let's add a `MaxOrReadableCount`

Reply:
Indeed, fixed.